connector_salesforce/README.rst

```diff
@@ -0,0 +1,247 @@
+Salesforce Connector
+====================
+
+This module allows to import and export various models between
+Odoo and Salesforce.
+
+It provides the foundations to create or extend various synchronizations.
+
+By default following synchronisations are supported out of the box:
+
+ - Import of Account and contact into Odoo Partners. Salesforce is the master.
+   Parent relation of Sales force account is not supported at this time.
+ - Import or export of Product. ERP or Salesforce can be master. Bidirectional synchronisation is not supported and a master must be set.
+ - Import of Salesforce priceBook entries into Odoo pricelist version. Salesforce is the master.
+ - Import of won opportunities into Odoo. Salesforce is the master.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * install simplesalesforce
+
+    `pip install simple-salesforce`
+
+ * install requests
+
+    `pip install requests`
+
+
+Configuration
+=============
+
+Configure Connection With Salesforce
+------------------------------------
+
+There are 2 supported ways to access a Salesforce instance with the Salesforce connector
+
+Oauth2 flow
+***********
+
+In Salesforce
+#############
+
+In order to use it you have to add a remote application in Salesforce administration
```

Comment:
First, you need to add a remote application in Salesforce administration
and enable Oauth2 login. The application should preferably be renamed `odoo_connector`.